feat(edge-services): update doc following migration (#6556)
scaleway/docs-content: pages/edge-services/reference-content/understanding-ga.mdx

Edge Services is billed via **subscription**. Three different subscription plans are available, each including different amounts of monthly usage for **cache size** and **number of pipelines**. If your consumption of Edge Services in a given month exceeds the cache size and number of pipelines included in your subscription, you are billed for this extra usage on top of your plan.

<Message type="tip">

> An Edge Services pipeline consists of a backend for which Edge Services requests and caches content, and an endpoint from which this content is served via Edge Services. In the case of Object Storage during Public Beta, a pipeline corresponded to a bucket Edge Services was enabled on. If Edge Services was enabled on three different buckets, this equaled three pipelines. In the case of Load Balancer, pipelines were already an explicit concept in the console.

Reviewer: ```suggestion
An Edge Services pipeline consists of a backend for which Edge Services requests and caches content, and an endpoint from which this content is served via Edge Services. In the case of Object Storage, during Public Beta, a pipeline corresponded to a bucket on which Edge Services was enabled. If Edge Services was enabled on three different buckets, this equaled three pipelines. In the case of Load Balancer, pipelines were already an explicit concept in the console.
```

> An Edge Services pipeline consists of a backend for which Edge Services requests and caches content, and an endpoint from which this content is served via Edge Services. In the case of Object Storage during Public Beta, a pipeline corresponded to a bucket Edge Services was enabled on. If Edge Services was enabled on three different buckets, this equaled three pipelines. In the case of Load Balancer, pipelines were already an explicit concept in the console.

Reviewer 2: ```suggestion
An Edge Services pipeline consists of a backend, for which Edge Services requests and caches content, and an endpoint from which this content is served via Edge Services. Previously, during the Object Storage Public Beta, a pipeline corresponded to a bucket that Edge Services was enabled on. If Edge Services was enabled on three different buckets, this meant three pipelines. With Load Balancer, pipelines were already an explicit concept in the console.
```
